glue_jupyter/bqplot/common/tools.py

```diff
@@ -25,6 +25,24 @@ def deactivate(self):
         self.viewer._mouse_interact.next = None
 
 
+class BqplotSelectionTool(InteractCheckableTool):
+    """Disables all other tools when activated."""
+
+    def activate(self):
+        with self.viewer._output_widget:
+            self.interact.selected_x = None
+            self.interact.selected_y = None
+
+        for viewer in self.viewer.session.application.viewers:
+            if viewer is not self.viewer:
+                viewer.toolbar.active_tool = None
+
+        # Jumps back to "create new"
+        self.viewer.session.edit_subset_mode.edit_subset = None
```

Comment:
Okay, `None` does work as you said. It is weird that `[]` is allowed but doesn't do anything. 🤷